src/components/entries/TextField.js

```diff
@@ -106,7 +112,10 @@ export default function TextfieldEntry(props) {
       error ? 'has-error' : '')
     } data-entry-id={ id }>
       <Textfield id={ id } label={ label } value={ value } onInput={ handleChange } debounce={ debounce } disabled={ disabled } />
-      { description && <div class="bio-properties-panel-description">{ description }</div> }
+      { (description || contextDescription) &&
```

Comment:
As this duplicates across all components we could consider moving it out into a separate `<Description />` component.

Reply:
Good point!
https://github.com/bpmn-io/properties-panel/pull/122/commits/5a32bd0185beece56ec524b8b283e0e978305987